include/curl/curl.h

```diff
@@ -2800,6 +2800,14 @@ typedef struct {
 #define CURL_VERSION_ALTSVC       (1<<24) /* Alt-Svc handling built-in */
 #define CURL_VERSION_HTTP3        (1<<25) /* HTTP3 support built-in */
 
+
+/* CURL_WINDOWS_CA_STORE contain a magic value for cacert with
+ curl compiled for MS-Windows using OpenSSL for using Windows CA Store
+    to verify peer
+  as suggested in https://curl.haxx.se/mail/lib-2018-09/0039.html */
+#define CURL_WINDOWS_CA_STORE     "\\\\\\WINDOWS_CA_STORE"
```

Comment:
The number of backslashes seems a bit weird, especially considering escaping. Maybe think of something more clear?

Reply:
there is 3*2=6. backslash
on windows a path beginning with two backslash can be a network path
three baskslash is impossible so there is no conflict
https://en.wikipedia.org/wiki/Path_(computing)#Uniform_Naming_Convention )

Reply:
If you'd use three forward-slashes instead, you'd avoid a lot of escaping problems and still be pretty safe, won't you? I think the main problem is here if we expect command line users to be able to set this.

Reply:
ok, I just replaced backslash by forward slash